src/PrizePool.sol

```diff
@@ -337,10 +341,10 @@ contract PrizePool {
         }
     }
 
-    /// @notice Completes the current prize period and starts the next one, updating the number of tiers, the winning random number, and the prize pool reserve
+    /// @notice Allows the Manager to complete the current prize period and starts the next one, updating the number of tiers, the winning random number, and the prize pool reserve
     /// @param winningRandomNumber_ The winning random number for the current draw
     /// @return The ID of the completed draw
-    function completeAndStartNextDraw(uint256 winningRandomNumber_) external returns (uint32) {
+    function completeAndStartNextDraw(uint256 winningRandomNumber_) external onlyManagerOrOwner returns (uint32) {
```

Comment:
Shouldn't it only be the draw mechanism that can start draws?
Something like the DrawBeacon contract.

Reply:
That would be the "manager" in this case.  I'm happy to make that change.